tests/fuzz/testdata/FuzzConfigValidation2/4534596883578880

```diff
@@ -0,0 +1 @@
+�{"http":  [  {  },  null, {"match":  [{},   {"        ":{}}, {" ":{}},null ,{"    ":{}}, {" ":{}},null  ]}]}                
```

Comment:
is that a special character at the very front?

Reply:
Yes, the test cases come from oss-fuzz.com, just copying them directly as regression tests. I didn't think up this abomination :stuck_out_tongue: